src/renderer/windows/settings/Settings.vue

```diff
@@ -396,6 +399,14 @@ window.ytmd.handleUpdateDownloaded(() => {
             <p class="discord-failure">Discord connection could not be established after 30 attempts</p>
             <button @click="restartDiscordPresence">Retry</button>
           </div>
+          <YTMDSetting 
+          v-model="discordPresenceListening" 
+          type="checkbox" name="Discord presence 
+          listening" @change="settingsChanged"/>
+          <div 
+          v-if="discordPresenceListening" 
+            class="setting indented">
+          </div>
```

Comment:
Missing Indentation
And please do not break inside of a Attribute.

I feel like that prettier was not run from this, and should run automatically before submitting a commit?

Reply:
Wdym, you want me to refactor the code or something? I believe I fixed it up a little bit tho